Source/ACE.Entity/Enum/PositionType.cs

```diff
@@ -174,6 +174,7 @@ public enum PositionType
         /// needs research
         /// </summary>
         RelativeDestination = 26,
-        TeleportedCharacter = 27
+        TeleportedCharacter = 27,
+        PoiDestination = 28
```

Comment:
It will require some way to get poi portals a new type seemed easier with the orm there are probably other ways to do it.